akka-actor/src/main/scala/akka/dispatch/AbstractDispatcher.scala

```diff
@@ -54,19 +54,25 @@ private[akka] trait LoadMetrics { self: Executor ⇒
  * INTERNAL API
  */
 private[akka] object MessageDispatcher {
-  val UNSCHEDULED = 0 //WARNING DO NOT CHANGE THE VALUE OF THIS: It relies on the faster init of 0 in AbstractMessageDispatcher
+  val UNSCHEDULED = 0
+  //WARNING DO NOT CHANGE THE VALUE OF THIS: It relies on the faster init of 0 in AbstractMessageDispatcher
   val SCHEDULED = 1
   val RESCHEDULED = 2
 
   // dispatcher debugging helper using println (see below)
   // since this is a compile-time constant, scalac will elide code behind if (MessageDispatcher.debug) (RK checked with 2.9.1)
-  final val debug = false // Deliberately without type ascription to make it a compile-time constant
+  final val debug = false
+  // Deliberately without type ascription to make it a compile-time constant
   lazy val actors = new Index[MessageDispatcher, ActorRef](16, _ compareTo _)
+
   def printActors(): Unit =
     if (debug) {
       for {
         d ← actors.keys
-        a ← { println(d + " inhabitants: " + d.inhabitants); actors.valueIterator(d) }
```

Comment:
You know that you can add a query parameter '?w=1' to the diff view to suppress whitespace changes, right? The only downside is that for comments you'll have to use a pristine tab.